src/lib/swapper/manager/SwapperManager.test.ts

```diff
@@ -242,20 +244,19 @@ describe('SwapperManager', () => {
       const swappers = await swapperManager.getSwappersWithQuoteMetadata({
         ...quoteInput,
         feeAsset: ETH,
-        buyAssetFiatRate: '1300',
-        sellAssetFiatRate: '0.04',
-        feeAssetFiatRate: '1300',
+        cryptoMarketDataById,
+        assetsById,
       })
       const expectedSwappers: SwapperWithQuoteMetadata[] = [
         {
           swapper: zrxEthereumSwapper,
           quote: bestTradeQuote,
-          inputOutputRatio: 0.5030214870725355,
+          inputOutputRatio: 0.5763658207357724,
```

Comment:
these ratios are different because the formula used to compute them has changed in  `getRatioFromQuote` from 

```
ratio = (receive_amount_fiat - receive_fee_fiat) / (send_amount_fiat + network_fee_fiat + send_fee_fiat)
```

to 

```
ratio = receive_amount_usd / (send_amount_usd + network_fee_usd + fees_total_usd)
```